setup.py

```diff
@@ -62,6 +62,6 @@
             language="c",
         ),
     ],
-    python_requires=">=3.7.*",
+    python_requires=">=3.7",
```

Comment:
@koubaa This was causing the following issue:
![image](https://user-images.githubusercontent.com/87315832/210372929-f08d5ce0-7421-4e7e-b970-204421acb3a9.png)
See the [action](https://github.com/pyansys/hollerith/actions/runs/3828897149/jobs/6515094247)